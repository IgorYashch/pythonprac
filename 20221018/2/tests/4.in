:f x y a log(x)+log(y)-a
:g x a b x**a+x**b
f 10 100 7
g 7 1 4
g 1 1 1
g 3 0 1
f 1 1 33
quit "{}-:-{}"