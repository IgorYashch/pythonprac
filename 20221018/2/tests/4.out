-0.0922447210178623
2408
2
4
-33.0
3-:-8
